### PR TITLE
Update cluster example

### PR DESCRIPTION
**Pull request by sarahkleins**
## Launch Checklist

- [x] briefly describe the changes in this PR
- [ ] @mollymerp for review

This PR updates the cluster example to use data-driven styling based on the point_count property.

screenshot 2017-04-10 14 35 48

### CODE REVIEW

**Comment by mourner** [comment]
Great to see this done! The colors are a bit hard to discern now — can we use more colors in the stops for more variety? We could use http://colorbrewer2.org/ to find a good scheme (e.g. "diverging" type with at least 5 colors).  The `colorSpace` property can also be helpful — using `lab` color space will make the color transitions more natural.

**Comment by incanus** [comment]
See also https://github.com/mapbox/ios-sdk-examples/pull/68 for some recent work on this on iOS. Color scheme works pretty ok there, I think?

**Comment by sarahkleins** [comment]
Thanks @incanus @mourner - I tried out the diverging type on http://colorbrewer2.org/ but couldn't really find a color scheme that worked -  I ended up going with the colors from the original example (blue, yellow, pink). @mollymerp could I get your 👀 for another round of review?

File: docs/_posts/examples/3400-01-02-cluster.html [hunk 1]
```diff
@@ -31,56 +30,55 @@
...
-        "type": "symbol",
-        "source": "earthquakes",
-        "filter": ["!has", "point_count"],
-        "layout": {
-            "icon-image": "marker-15"
+        id: "clusters",
+        type: "circle",
+        source: "earthquakes",
+        filter: ["has", "point_count"],
+        colorSpace: "lab",
```
Comment (mollymerp) [inline docs/_posts/examples/3400-01-02-cluster.html]: I don't think you need `colorSpace` unless you're using an exponential function (colorSpace is used for interpolation, and `categorical` functions don't interpolate color)

File: docs/_posts/examples/3400-01-02-cluster.html [hunk 2]
```diff
@@ -31,56 +30,55 @@
...
+                    [20, "#f1f075"],
+                    [150, "#f28cb1"],
+                ]
+            },
+            "circle-radius": {
+                property: "point_count",
+                type: "interval",
+                stops: [
+                    [0, 20],
+                    [1000, 40]
```
Comment (mollymerp) [inline docs/_posts/examples/3400-01-02-cluster.html]: color and radius should have the same number of stops with the same property values

**Comment by mollymerp** [comment]
Just a few more comments @sarahkleins - thanks for the quick follow up! I also liked the stroke + white text, but I'll leave styling choices up to you 🎨

**Comment by sarahkleins** [comment]
thanks @mollymerp! :pray: updates complete 👀

**Pull request merged**